.github/workflows/osrm-backend.yml

```diff
@@ -400,7 +400,7 @@ jobs:
 
     - name: Install dev dependencies
       run: |
-        python3 -m pip install conan==1.53.0
+        python3 -m pip install conan==1.59.0
```

Comment:
Not sure, may be it is better to just always install the latest 1.x

Reply:
IMO `<2.0.0` is best. It's how I do it on Valhalla, never had problems. In fact we were always installing the latest, which bit us a few days ago when 2.x was released

Reply:
Agree :+1: